concrete/blocks/image/controller.php

```diff
@@ -152,6 +154,24 @@ public function getJavaScriptStrings()
     }
 
     /**
+     * Check if the file is an SVG
+     *
+     * @param  \Concrete\Core\Entity\File\File $f The file object for the selected image
+     *
+     * @return bool|null Return true if the file is an SVG, null otherwise
+     */
+    public function isSVG($f)
```

Comment:
I feel like this shouldn't be in this file...

What if we added a new method to the file Object that is `isCroppable()` or `isResizable()`?